test/helm-chart/README.md

```diff
@@ -0,0 +1,7 @@
+# Testing Helm chart
+
+This Helm chart was created by running:
+
+```sh
+helm create helm-chart
+```
```

Comment:
This file could probably be removed. Seems to generate false hope of having lots of usefull stuff to read. This does not say much. If there is a link to the helm docs in the main readme that probably a better place to go.

Reply:
I've updated the file. Hopefully it's more explanatory now.

I like having this file here just to clear up any possible confusion, so noone thinks wharf-cmd actually got its own Helm chart.